.github/workflows/backup.yaml

```diff
@@ -1,55 +1,76 @@
 name: Backup Routine
+
 on:
   workflow_dispatch:
   schedule:
     # Runs at 04:00 UTC on the 1st and 17th of every month
     - cron: "0 4 */16 * *"
+
 jobs:
   backup-ba-dataset:
     runs-on: ubuntu-latest
     name: Backup ba dataset
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Bruk yarn cache
+        uses: actions/setup-node@v3
+        with:
+          node-version: latest
+          cache: 'yarn'
```

Comment:
Tjener ~30 sekunder med cache minus ~15 sekunder for å hente opp cachen. Dersom den ikke finner cachen får vi et tap på over 1 minutt som nesten er en dobling av tiden backup tar. 

Dette er ikke prosjektet der vi får mest ut av cachen siden vi bare bruket én pakke. Tenker uansett at det er gøy å ha med, og jeg tror vi over tid sparer sekunder på dette.

Reply:
Kanskje ikke veldig viktig med.. 
Hva brukes cachen egentlige til her? Her brukes vel cachen til selve prosjektet, som inneholder mye annet? 
Hva skjer hvis man skipper setup-node helt? Er det då det tar (30s) lengre tid å installere sanity?

Reply:
(Cachen slettes hvis den ikke er brukt etter 7 dager)

Reply:
>Hva brukes cachen egentlige til her?
Vi bruker cachen når vi laster ned `sanity/cli` i neste steg: 
![image](https://github.com/navikt/familie-sanity-brev/assets/17828446/0b98c4cb-3d2a-4498-9b21-9d058957b434)

> Her brukes vel cachen til selve prosjektet, som inneholder mye annet?
Det kan stemme. Er det et problem?

>Hva skjer hvis man skipper setup-node helt
Da må vi installere uten cache. Under er det noen eksempler, og i det førstet kjører vi uten setup-node steget:
* Uten cache 1m 36s: https://github.com/navikt/familie-sanity-brev/actions/runs/4957419271/jobs/8869065883
* Med cache, men cache miss 2m 5s: https://github.com/navikt/familie-sanity-brev/actions/runs/4957520902/jobs/8869297434
* Med cache, cache hit 1m 7s: https://github.com/navikt/familie-sanity-brev/actions/runs/4957542836/jobs/8869348346

Reply:
>Cachen slettes hvis den ikke er brukt etter 7 dager, så cachen vil nog ikke påvirke noe med tanke på at jobbet kjøres 2 ganger i måneden :)

👀

Reply:
Sletta `setup-node`-steget her: [Fjern bruk av cache siden det uansett slettes etter 7 dager](https://github.com/navikt/familie-sanity-brev/pull/390/commits/c6b8155e3e4d9d87993ec639b60136fc11038792)